docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -0,0 +1,283 @@
+.. _intervention_wasting_treatment:
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1
+  ---------------
+
+  Section Level 2
+  +++++++++++++++
+
+  Section Level 3
+  ~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+===============================================
+Treatment and management for acute malnutrition 
+===============================================
+
+This documentation focuses on treatment and management of acute malnutrition in Ethiopia based on 2019 National Guideline for the Management of Acute Malnutrition. [EMOH]_
+
+The prevalence of stunting in Ethiopia has declined from 58% in 2000 to 38% in 2016. However, the prevalence of wasting has remained fairly static at 12% in 2000 and 10% in 2016. To address malnutrition in all its forms, the Government is applying two programmatic approaches. The first focuses on increasing access to and availability of food through improved economic growth, better agricultural production systems along with promotion of good nutrition practices and prevention of malnutrition. The second approach aims to strengthen early warning systems and timely emergency response, including wide-scale delivery of services for the management of acute
+malnutrition.
+
+The Federal Ministry of Health (FMOH) developed the first Protocol for the Management of Severe Acute
+Malnutrition (SAM) in 2007, and the Guideline for the Management of Moderate Acute Malnutrition (MAM)
+in 2012. This is the latest National Guideline for the Management of Acute Malnutrition in Ethiopia (2019). It includes the latest World Health Organisation (WHO) guidelines and recommendations, and emerging national and international evidence. It is also aligned to the National Nutrition Programme (NNP) II 2016-2020, the National Food and Nutrition Policy and the Health Sector Transformation Plan (HSTP) 2015/16 - 2019/20.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+  * - FMOH
+    - Federal ministry of health
+  * - SAM
+    - severe acute malnutrition 
+  * - MAM
+    - moderate acute malnutrition
+  * - DHS
+    - demographic health survey
+  * - IMNICI
+    - Integrated Management of Newborn and Childhood illness
+
+
+Intervention Overview
+---------------------
+
+This flow chart summarizes the core aspects of the care and treatment for SAM and MAM, and integration into the
+routine health system. 
+
+.. image:: flow_chart_management_of_acute_malnutrition
```

Comment:
can you add definitions for sc, otp, and tsfp?

Reply:
yes!!